### ISSUE TITLE
Scraping error: history.account-menu: expected 1 element, got 0

### ISSUE THREAD

**Issue by netmaniac** [jwilk/mbank-cli#101]
Starting from Sunday mbank-cli can not get account history:
```
./mbank-cli history -M *biezace
Scraping error: history.account-menu: expected 1 element, got 0 at ./mbank-cli line 347.
	main::unpack_list(ARRAY(0x5605056530d8), ARRAY(0x560505653000), "context", "history.account-menu") called at ./mbank-cli line 1284
	main::html_find(HTML::TreeBuilder=HASH(0x5605056532a0), "id", "MenuAccountsCombo", "context", "history.account-menu") called at ./mbank-cli line 2165
	main::_switch_history_account("document", HASH(0x560505674b40), "number", "XXXXXX") called at ./mbank-cli line 2207
	main::_do_history_account("document", HASH(0x560505674b40), "number", "XXXXX", "name", "mBiznes konto - biezace", "display_name", 1, ...) called at ./mbank-cli line 2143
	main::do_history("selection", ARRAY(0x560505100d78), "display_name", 1, "login", HASH(0x560505675740), "accounts", ARRAY(0x560505657618), ...) called at ./mbank-cli line 3662
	main::main() called at ./mbank-cli line 3666
```

Are others experiencing that issue or this is something local for me?

**Comment by aiv** [comment]
Same story on my side, from Saturday mBank maintenance.

**Comment by jwilk** [comment]
I guess like the old history API has been finally removed.

You can use the `history2019` command for now; see bug #84.

**Comment by chilek** [comment]
The same problem in my installation. history2019 works.

**Comment by jwilk** [comment]
I suppose instead of switching to the new API, we could try to parse the CSV exports.
AFAICS the exports have all the information we printed in the past.

**Comment by jwilk** [comment]
> I suppose instead of switching to the new API, we could try to parse the CSV exports.

I took a stab at this: see the `wip/csv` branch.
At the moment it requires a fairly new version (≥ 1.90, I think) of [Text::CSV](https://metacpan.org/release/Text-CSV).
I hope to clean it up tomorrow.

> AFAICS the exports have all the information we printed in the past.

Well, almost all. Operation identifiers are missing. ☹
So for now `--with-id` will remain broken.

**Comment by jwilk** [comment]
I've got rid of the Text::CSV dependency and merged the CSV parsing code to master (379335154b34c67c42122876158a52461dc0e3bb).
